lib/active_fedora/file.rb

```diff
@@ -196,7 +216,7 @@ def uploaded_file?(payload)
       end
 
       def local_or_remote_content(ensure_fetch = true)
```

Comment:
Would it make sense to settle on the word "external" instead of "remote" for this method (and any others that may be applicable)? We could `alias :local_or_remote_content, :local_or_external_content`, and add a deprecation warning.